frontend/public/components/dashboards-page/overview-dashboard/top-consumers-card.tsx

```diff
@@ -0,0 +1,167 @@
+import * as React from 'react';
+import { connect } from 'react-redux';
+import { get } from 'lodash';
+import { Col } from 'patternfly-react';
+
+// TODO Use PF4 dropdown
+// import { DROPDOWN } from '../../Form/constants';
+import {
+  DashboardCard,
+  DashboardCardBody,
+  DashboardCardHeader,
+  DashboardCardTitle,
+  DashboardCardHelp,
+} from '../../dashboard/dashboard-card';
+import {
+  formatCpu,
+  getMetric,
+  getMetricConsumers,
+  formatBytesWithUnits,
+  ConsumerItem,
+  ConsumersResults,
+  ConsumersFilter
+} from '../../dashboard/top-consumers-card';
+import {
+  BY_CPU,
+  BY_NETWORK,
+  BY_STORAGE,
+  BY_MEMORY,
+  PODS_AND_VMS,
+  INFRASTRUCTURE,
+  NODES,
+  WORKLOADS,
+  CPU_DESC,
+  MEMORY_DESC,
+  STORAGE_DESC,
+  NETWORK_DESC,
+} from '../../dashboard/top-consumers-card/strings';
+import { DashboardItemProps, withDashboardResources } from '../with-dashboard-resources';
+import { RootState } from '../../../redux';
+import { featureReducerName } from '../../../reducers/features';
+import { FLAGS } from '../../../const';
+
+const metricTypes = [{ name: PODS_AND_VMS, description: WORKLOADS }, { name: NODES, description: INFRASTRUCTURE }];
+
+const sortByOptions = [
+  { name: BY_CPU, description: CPU_DESC },
+  { name: BY_MEMORY, description: MEMORY_DESC },
+  { name: BY_STORAGE, description: STORAGE_DESC },
+  { name: BY_NETWORK, description: NETWORK_DESC },
+];
+
+const mapStateToProps = (state: RootState) => ({
+  openShiftFlag: state[featureReducerName].get(FLAGS.OPENSHIFT),
+});
+
+const _TopConsumersCard: React.FC<TopConsumerProps> = ({
```

Comment:
Current console convention is to put the `_` at the end.

```suggestion
const TopConsumersCard_: React.FC<TopConsumerProps> = ({
```